locale/en/docs/guides/diagnostics-flamegraph.md

```diff
@@ -39,7 +39,7 @@ Now let's get to work.
     ```
 
 4. disregard warnings unless they're saying you can't run perf due to missing packages; you may get some warnings about not being able to access kernel module samples which you're not after anyway.
-5. Run `perf script > perfs.out` to generate the data file you'll visualize in a moment. It's useful to [apply some cleanup](#filtering-out-node-internal-functions) for a more readable graph
+5. Run `perf script > perfs.out` to generate the data file you'll visualize in a moment. It's useful to [apply some cleanup](#filtering-out-node-js-internal-functions) for a more readable graph
```

Comment:
Looks like the fix for the built/metalsmith version isn't the same as GitHub (which has it as `nodejs` with no dash).
Not a big deal since I think it's just broken for both ways right now, but might be an issue if adding https://github.com/remarkjs/remark-validate-links